data/phi0474/phi001/phi0474.phi001.perseus-eng2.xml

```diff
@@ -5,11 +5,9 @@
     <fileDesc>
       <titleStmt>
         <title>For P. Quinctius</title>
-        <title>For Quinctius, Sextus Roscius, Quintus Roscius, against Quintus Caecilius, and against
-          Verres</title>
-        <title type="sub">Machine readable text</title>
-        <author>M. Tullius Cicero</author>
-        <editor role="translator" n="Yonge">C. D. Yonge</editor> <sponsor>Perseus Project, Tufts University</sponsor>
+        <author>Cicero</author>
```

Comment:
This name should match the name you want as the canonical name. So it could be Cicero or M. Tullius Cicero — I don't know if we chose one over the other. Usually, I have the catalog name here.